parquet/src/main/java/org/apache/iceberg/parquet/ParquetVariantVisitor.java

```diff
@@ -21,17 +21,14 @@
 import java.util.List;
 import org.apache.iceberg.relocated.com.google.common.base.Preconditions;
 import org.apache.iceberg.relocated.com.google.common.collect.Lists;
+import org.apache.iceberg.variants.Variant;
 import org.apache.parquet.schema.GroupType;
 import org.apache.parquet.schema.LogicalTypeAnnotation.ListLogicalTypeAnnotation;
 import org.apache.parquet.schema.PrimitiveType;
 import org.apache.parquet.schema.PrimitiveType.PrimitiveTypeName;
 import org.apache.parquet.schema.Type;
 
 public abstract class ParquetVariantVisitor<R> {
-  static final String METADATA = "metadata";
-  static final String VALUE = "value";
-  static final String TYPED_VALUE = "typed_value";
```

Comment:
I don't think these changes are adding much value. I'd revert them and not add values to the public API.